docs/src/components/App/Header/Header.js

```diff
@@ -60,7 +60,9 @@ class Header extends Component {
   };
 
   handleDocumentKeydown = event => {
-    if (event.key === '/' && event.target.id !== SEARCH_BAR_ID) {
+    const isInputField =
+      ['INPUT', 'TEXTAREA'].indexOf(event.target.tagName) !== -1;
```

Comment:
Personal preference maybe, but I'd prefer to see this as a case insensitive regex.